QUARKUS-1936.md

```diff
@@ -0,0 +1,31 @@
+# QUARKUS-1936 Certify RHBQ on Eclipse Adoptium Builds
+
+This document is a high level test plan for certification of Red Hat build of Quarkus on Temurin JVM.
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* Temurin JVM 11 and 17 need to be added as JDKs to Quarkus QE Jenkins
+  * Temurin JVM 11 and 17 seem to be updated timely on `/qa/tools` mount point, so that will be used for the Elektra
+    time frame.
+* Temurin JVM 11 and 17 have to be added as JVMs to Polarion
+* Bare metal and OpenShift jobs in extended platform testing need to be extended to be ran with both Temurin 11 and 17.
```

Comment:
How will be OpenShift testing affected? Do we assume new base image instead of OpenJDK one? 
Or just app compiled using Temurin but running in OpenJDK based pod?

TBH I don't see much value in regular OpenShift testing with Temurin if there is no Temurin base image we support

Reply:
https://issues.redhat.com/browse/QUARKUS-1936

> Note that there are no UBI Java images based on Adoptium so we are only targeting bare metal at this point.